api.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+)
+
+type RecurseAPI struct {
+	rcAPIURL string
+}
+
+type RecurserProfile struct {
+	Stints []Stint
+}
+
+type Stint struct {
+	In_progress bool
+}
+
+func (ra *RecurseAPI) userIsCurrentlyAtRC(accessToken string, email string) bool {
+	currentlyAtRC := false
+
+	resp, err := http.Get(ra.rcAPIURL + "/profiles/" + email + "?access_token=" + accessToken)
+
+	if err != nil {
+		log.Printf("Got the following error while checking if the user was active at RC: %s\n", err)
+		return currentlyAtRC
+	}
+
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+
+	//Parse the json response from the API
+	profile := RecurserProfile{}
+	json.Unmarshal([]byte(body), &profile)
+
+	log.Println("Recurser Profile: ", profile)
+
+	currentlyAtRC = profile.Stints[0].In_progress
+
+	log.Println("in_progress:", currentlyAtRC)
+
+	return currentlyAtRC
+}
+
+func (ra *RecurseAPI) getCurrentlyActiveEmails(accessToken string) []string {
+	//TODO, batch the API call since the limit is 50 results per
```

Comment:
😓 

Right now we have fewer than 50 folks at RC, but that's not a good assumption to use moving forward. Will get to fixingthis shortly.